index.js

```diff
@@ -22,4 +22,38 @@ addTaskButton.addEventListener("click", () => {
   });
 
   console.log(tasks);
+
+  taskTable = document.getElementById("task-table");
```

Comment:
Is this a new variable? if yes. why the definition does not have `const`? And please move this to the top. Where you defined all variables.

Reply:
oh,that was a foolish mistake I've done.I'll fix it now.